src/explore-education-statistics-common/src/modules/release/components/__tests__/NationalStatisticsSection.test.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import NationalStatisticsSection from '@common/modules/release/components/NationalStatisticsSection';
+
+describe('NationalStatisticsSection', () => {
+  test('renders', () => {
+    render(<NationalStatisticsSection />);
+
+    expect(
+      screen.getByRole('link', {
+        name: 'Office for Statistics Regulation',
+      }),
+    ).toBeInTheDocument();
```

Comment:
should we also check that the link is correct here? The same as you have done for the other ones below. Or maybe we duplicate the test somewhat in the test `includes the OSR guidance text` below since this test is purely to test it `renders`

Reply:
This is just a basic 'check it renders at all' test so I don't think it needs to be that specific, it just needs to check that something we expect renders.

Reply:
agreed, I was more just wondering if we'd want to do this check in the other test. But if we don't want to check absolutely everything then I'm happy :)